### PR TITLE
BRITECH-119 Change lastcontacted endpoint to last-contacted

### PR DESCRIPTION
**Pull request by dwybourn**

### CODE REVIEW

File: Speaker-comms/LastContactedController.fs [hunk 1]
```diff
@@ -5,6 +5,7 @@ open SpeakerComms.LastContactedService
 open System.Net.Http
 open Serilog
 
+[<Route("last-contacted")>]
```
Comment (lclarksl) [inline Speaker-comms/LastContactedController.fs]: Are we changing this to last-contact?

**Comment by csscottlogic** [comment]
:shipit:

File: Speaker-comms/LastContactedController.fs (hunk 1 above)
Comment (csscottlogic) [inline Speaker-comms/LastContactedController.fs]: I'd vote for that

**Comment by lclarksl** [comment]
Decide on last-contacted/last-contact then :shipit:

**Comment by dwybourn** [comment]
Don’t you mean just last-contact?

**Comment by csscottlogic** [comment]
I think last-contacted / last-contact indicates a choice @dwybourn . Let's go with last-contact? We'll need to change the API definition.

**Pull request merged**